docs/en/query_language/functions/other_functions.md

```diff
@@ -720,34 +720,101 @@ Result:
 └───────────────────────────────┘
 ```
 
-## filesystemAvailable {#function-filesystemavailable}
+## filesystemAvailable {#filesystemavailable}
 
-Returns the amount of remaining space in the filesystem where the files of the databases located. See the [path](../../operations/server_settings/settings.md#server_settings-path) server setting description.
+Returns amount of remaining space on the filesystem where the files of the databases located. It is always smaller than total free space ([filesystemFree](#filesystemfree)) because some space is reserved for a `root` user.
+
+**Syntax**
 
 ```sql
-filesystemAvailable()
+filesystemAvailable();
 ```
 
-**Returned values**
+**Returned value**
 
-- Amount of remaining space in bytes.
+- The amount of remaining space for writing data in bytes.
 
 Type: [UInt64](../../data_types/int_uint.md).
 
 **Example**
 
+Query:
+
 ```sql
-SELECT filesystemAvailable() AS "Free space", toTypeName(filesystemAvailable()) AS "Type"
+SELECT filesystemAvailable() AS "Available space", toTypeName(filesystemAvailable()) AS "Type";
 ```
+
+Result:
+
+```text
+┌─Available space─┬─Type───┐
+│     34966368256 │ UInt64 │
+└─────────────────┴────────┘
+```
+
+## filesystemFree {#filesystemfree}
+
+Returns total amount of the free space on the filesystem where the files of the databases located. Consist of both space reserved for a `root` user and space that actual available for data.
```

Comment:
These functions also need a paragraph on how they work in conjunction with multi-tier storage feature.

Reply:
Do we have to do this now? When we have such documentation https://clickhouse.yandex/docs/en/query_language/functions/type_conversion_functions/#touint-8-16-32-64-ornull